src/core/include/units/magnitude.h

```diff
@@ -0,0 +1,331 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2018 Mateusz Pusz
```

Comment:
Is this right?  I note that it is not 2018, and I am not Mateusz Pusz, but I tried following the style in the other files.

Reply:
Just checking in again about this copyright notice question.

Reply:
I just copy the same to all the files and I noticed that other contributors do the same.

I do not mind changing this approach but frankly speaking, I do not have a good idea how to do it. All of the contributors are enumerated automatically by GitHub already. I also listed the most active contributors in the citation file (at least 50 commits).

Please let me know in case you have some suggestions.